features/config_json_formatter.feature

```diff
@@ -0,0 +1,43 @@
+@test_unit @config
+Feature:
+
+  SimpleCov::Formatter::JSONFormatter is one of the
+  formatters included by default, useful for exporting
+  coverage results in JSON format.
+
+  Background:
+    Given I'm working on the project "faked_project"
+  Scenario: With JSONFormatter
+    Given SimpleCov for Test/Unit is configured with:
+      """
+      require 'simplecov'
+      require 'simplecov_json_formatter'
+      SimpleCov.formatter = SimpleCov::Formatter::JSONFormatter
+      SimpleCov.at_exit do
+        puts SimpleCov.result.format!
+      end
+      SimpleCov.start do
+        add_group 'Libs', 'lib/faked_project/'
+      end
+      """
+
+    When I successfully run `bundle exec rake test`
+    Then a JSON coverage report should have been generated in "coverage"
+    And the output should contain "JSON Coverage report generated"
+
+  Scenario: When CC_TEST_REPORTER_ID is set in the environment
+    Given SimpleCov for Test/Unit is configured with:
+      """
+      ENV['CC_TEST_REPORTER_ID'] = "9719ac886877886b7e325d1e828373114f633683e429107d1221d25270baeabf"
```

Comment:
move into invoking command and make id more generic/show it's fake (comment for myself when fixing it up)